### ISSUE TITLE
[🐛 Bug]: import error on 9.1.3

### ISSUE THREAD

**Issue by brickfungus** [webdriverio/webdriverio#13747]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

9.1.3

### Node.js Version

20.17.0

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
capabilities: [
    {
      browserName: 'chrome',
      browserVersion: 'stable',
      acceptInsecureCerts: true,
      'goog:chromeOptions': {
        args: [
          '--no-sandbox',
          '--headless=new',
          '--disable-gpu']
      },
    },
    {
      browserName: 'firefox',
      acceptInsecureCerts: true,
      'moz:firefoxOptions': {
        args: ['-headless', '--no-sandbox', '--disable-gpu']
      }
    },
  ],
```

### What happened?

Trying to run wdio after upgrading to 9.1.3 getting this error immediately 
```bash
PS C:\Users\josh\Documents\git\wdio-video-reporter> npm run demo:mocha
Debugger attached.

> wdio-video-reporter@6.0.1 demo:mocha
> npx wdio run demo/wdio.conf.ts

Debugger attached.
Debugger attached.
Waiting for the debugger to disconnect...
file:///C:/Users/josh/Documents/git/wdio-video-reporter/node_modules/htmlfy/src/minify.js:2
import { hasHtml } from "./utils.js"
         ^^^^^^^
SyntaxError: The requested module './utils.js' does not provide an export named 'hasHtml'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:146:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:229:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:473:24)
    at async file:///C:/Users/josh/Documents/git/wdio-video-reporter/node_modules/@wdio/cli/bin/wdio.js:15:17

Node.js v20.17.0
Waiting for the debugger to disconnect...
Waiting for the debugger to disconnect...
```

### How to reproduce the bug.

install 9.1.3 and try to run wdio

### Relevant log output

```typescript
doesn't even log
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
Duplicate of https://github.com/j4w8n/htmlfy/issues/7

**Comment by j4w8n** [comment]
I'm sorry about that @christian-bromann. Thought I had tested before release, but something got missed. Vitest tests are now catching this scenario.

Fixed in htmlfy v0.3.2
